Tutorials/intro-to-simulation/intro-to-simulation.py

```diff
@@ -0,0 +1,125 @@
+"""
+intro_to_simulation.py: Demonstrate usage of the simulation feature
+Author: Tomer Feld - Quantum Machines
+Created: 26/06/2022
+
+"""
+import numpy as np
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm import SimulationConfig
+from qm import LoopbackInterface
+from qm.qua import *
+import matplotlib.pyplot as plt
+from configuration import *
+
+qmm = QuantumMachinesManager()
+
+##
+# ## Example 1 - Basic simulation, No inputs
+
+with program() as prog1:
+    a = declare(fixed)
+    c = declare(fixed, value=0.4)
+
+    with for_(a, 0.2, a < 0.9, a + 0.1):
+        play("const" * amp(a), "qe1")
+
+    save(c, "c")
```

Comment:
I think we should use the stream processing and (maybe with `with_timestamps()` and `save_all()`).
Do we encourage users to use this save method?